mailbox/api/src/main/java/org/apache/james/mailbox/MailboxListener.java

```diff
@@ -208,13 +209,17 @@ public MailboxEvent(MailboxSession session, MailboxPath path, MailboxId mailboxI
                 .map(MailboxSession::getUser)
                 .map(MailboxSession.User::getCoreUser)
                 .orElse(null);
+            this.sessionId = Optional.ofNullable(session)
+                .map(MailboxSession::getSessionId)
+                .orElse(0L);
```

Comment:
please don't use 0 as a special value

Reply:
I remove this orElse by using optional session id in https://github.com/linagora/james-project/pull/1980/commits/7b2709baa1cde1bb164fc746cd64c0bb2510fc32